projects/eslint-config/package.json

```diff
@@ -13,46 +13,44 @@
     "index.js"
   ],
   "dependencies": {
-    "eslint": "8.32.0",
-    "@typescript-eslint/eslint-plugin": "5.48.1",
-    "@typescript-eslint/parser": "5.48.1",
+    "eslint": "8.37.0",
+    "@typescript-eslint/eslint-plugin": "5.57.0",
+    "@typescript-eslint/parser": "5.57.0",
     "eslint-plugin-deprecation": "1.3.3",
-    "eslint-plugin-import": "2.27.4",
-    "eslint-plugin-jsdoc": "39.6.4",
+    "eslint-plugin-import": "2.27.5",
     "eslint-plugin-no-loops": "0.3.0",
     "eslint-plugin-no-secrets": "0.8.9",
     "eslint-plugin-no-unsanitized": "4.0.2",
     "eslint-plugin-prefer-arrow": "1.2.3",
     "eslint-plugin-promise": "6.1.1",
     "eslint-plugin-unused-imports": "2.0.0",
     "eslint-plugin-xss": "0.1.12",
-    "@angular-eslint/builder": "14.0.2",
-    "@angular-eslint/eslint-plugin": "14.0.2",
-    "@angular-eslint/eslint-plugin-template": "14.0.2",
-    "@angular-eslint/schematics": "14.0.2",
-    "@angular-eslint/template-parser": "14.0.2"
+    "@angular-eslint/builder": "14.4.0",
```

Comment:
Perhaps we should consider if we need to specify @angular-eslint/xxx dependencies or not. Points to consider

- Do we support single angular version?
- Is the major version of @angular-eslint is aligned with the major angular version (@angular-eslint packages v14 and @angular v14 are compatible)?
- How is this 'compatibility' implied to users of the library. (Angular v13 users can install the library and have potential issues - there is no peer dependency on angular major version.)

I've had a look at the similar project, https://github.com/Tinkoff/linters. They don't include the @angular-eslint dependencies.